### DISCUSSION TITLE
Support for color blind themes

### DISCUSSION THREAD

**Discussion by tklein1801** [community/community#6455] [Projects and Issues]
Due to some minor issues with colors I prefer to use the dark color blind theme which isn't still a little buggy.
![msedge_nn5PN9o42M](https://user-images.githubusercontent.com/38816229/137650067-b68f7b37-0f21-4b21-a392-81e7b73db973.png)
The header is in an darkish-theme but the font-color is dark too and the other content is in an light theme.

**Comment by mariorod** [comment]
Hi, thanks for reporting this. We are working on a fix and it should be in production in the coming week.

**Comment by tklein1801** [comment]
The dark colorblind theme isn't working anymore.

![msedge_tr20kljINY](https://user-images.githubusercontent.com/38816229/143777995-31b770c3-31ba-458f-a651-ee6fb3996ebc.png)

**Reply by azenMatt** [reply]
Thanks for reporting this @tklein1801 - we just pushed a fix

![image](https://user-images.githubusercontent.com/7584089/144230439-19b239ec-ee4a-4653-934f-04159ff5bb23.png)
